provider/cloudflare/cloudflare_test.go

```diff
@@ -1312,11 +1312,11 @@ func TestCloudflareComplexUpdate(t *testing.T) {
 			},
 		},
 		{
-			Name: "UpdateDataLocalizationRegionalHostname",
+			Name:   "UpdateDataLocalizationRegionalHostname",
```

Comment:
Please avoid unrelated changes. Consider filing a separate PR for linting instead